code/modules/hydroponics/seeds.dm

```diff
@@ -248,6 +248,10 @@
 		return initial(plant.name)
 	else
 		return "[initial(plant.name)] (renamed as [plantname])"
+
+/obj/item/seeds/proc/log_rename(mob/user, obj/item, type, input)
+	src.investigate_log("[key_name(user)] has changed [src] ([type]) into '[input]'", INVESTIGATE_BOTANY)
+	// log_game() might fit, but I wanted botany logs contained in a specific log file.
```

Comment:
This needs to be logged to permanent storage if we are logging it

Reply:
It is permanent logging. It's just categorised rather than all-in-one log_game()